packages/ui-kit/src/Panel/index.tsx

```diff
@@ -2,50 +2,76 @@ import { Card } from '../Card';
 import cn from 'clsx';
 import s from './style.module.css';
 import { ReactChild } from 'react';
-import { Close24 } from '@konturio/default-icons';
+import { Text } from '../Text';
+import { ChevronDown24, ChevronUp24 } from '@konturio/default-icons';
+import { Modal } from '../Modal';
 
 interface Panel extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement> {
   className?: string;
   header?: string | React.ReactChild | React.ReactChild[];
-  onClose?: React.MouseEventHandler<HTMLButtonElement>;
+  headerIcon?: React.ReactChild;
+  isOpen?: boolean;
+  onHeaderClick?: React.MouseEventHandler<HTMLDivElement>;
   customCloseBtn?: ReactChild;
   classes?: {
     header?: string;
+    headerTitle?: string;
     closeBtn?: string;
+    modal?: string
+  };
+  modal?: {
+    onModalClick: () => void
+    showInModal: boolean;
```

Comment:
it looks like these properties are optional also, as onModalClick in optional in <Modal>
```suggestion
    onModalClick?: () => void
    showInModal?: boolean;
```

Reply:
My intention is following:
either you don't use modal mode,
either you do - but then consider 2 things:
1) since Modal mode is always something you intend to close - strictly provide onClose callback (so there's no way to forget it)
2) Modal mode can be optional - so I'd rather ask specifically when to use it